webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+{% extends "crashstats_base.html" %}
+
+{% block page_title %}User Documentation{% endblock %}
+
+{% block site_css %}
+    {{ super() }}
+    {% stylesheet 'documentation' %}
+{% endblock %}
+
+{% block content %}
+
+<div id="mainbody">
+
+    <div class="page-heading">
+        <h2>User Documentation</h2>
+    </div>
+
+    <div class="panel">
+        <div class="body">
+            <ul class="links">
+                <li>Super Search</li>
+                <li>
+                    <a href="{{ url('documentation:supersearch_home') }}">Home</a>
+                </li>
+                <li>
+                    <a href="{{ url('documentation:supersearch_api') }}">API</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="panel">
+        <div class="body">
+            <table>
+                <caption>Summary</caption>
+                <tbody>
+                    <tr>
+                        <td>User Interface</td>
+                        <td><a href="{{ url('supersearch.search') }}">{{ full_url(request, 'supersearch.search') }}</a></td>
+                    </tr>
+                    <tr>
+                        <td>API endpoint</td>
+                        <td><a href="{{ url('api:model_wrapper', model_name='SuperSearch') }}">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</a></td>
+                    </tr>
+                    <tr>
+                        <td>Private endpoint</td>
+                        <td><a href="{{ url('api:model_wrapper', model_name='SuperSearchUnredacted') }}">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearchUnredacted') }}</a></td>
+                    </tr>
+                    <tr>
+                        <td>API Documentation</td>
+                        <td><a href="{{ url('documentation:supersearch_api') }}">{{ full_url(request, 'documentation:supersearch_api') }}</a></td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <h1 id="what-is-super-search">What is Super Search?</h1>
+
+            <p>Super Search is an interface to crash reports data. There is a <a href="{{ url('supersearch.search') }}">human-friendly interface</a> and a <a href="{{ url('api:documentation') }}#SuperSearch">public API</a>. This guide covers the API, but the interface can be used to easily query and visualize the data.</p>
+            <p>There are 3 key points to Super Search. First, it is a secured window on all the raw crash reports data we have. Second, it allows you to use almost every field as a filter, giving you great power to explore the huge data we have. And third, it has advanced aggregation features that can be used to extensively analyze a subset of data.</p>
+            <p>This documentation aims at extensively covering the public API.</p>
```

Comment:
Drop the word "public" because, the SuperSearch API supports privileged stuff, doesn't it? Ie. the UnredactedSuperSearch.